tests/acceptance/addon-dummy-generate-test.js

```diff
@@ -1,11 +1,10 @@
 'use strict';
 
-const co = require('co');
-const RSVP = require('rsvp');
 const ember = require('../helpers/ember');
 const fs = require('fs-extra');
+const util = require('util');
 const path = require('path');
-let remove = RSVP.denodeify(fs.remove);
+let remove = util.promisify(fs.remove);
```

Comment:
`fs-extra` supports Promise-style invocation by default now, so this wrapper is not needed anymore.